pkg/boot/netboot/ipxe/ipxe.go

```diff
@@ -116,12 +116,20 @@ func parseURL(name string, wd *url.URL) (*url.URL, error) {
 	return u, nil
 }
 
+func (c *parser) createInitRD(initrds []io.ReaderAt) error {
```

Comment:
Name this "createInitrd" to be consistent with the rest of the capitilization in this file.